<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilenameEdit</class>
 <widget class="QMainWindow" name="FilenameEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FilenameEdit</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-text-editor">
    <normaloff>filenameEdit.ico</normaloff>
    <normalon>filenameEdit.ico</normalon>
    <disabledoff>filenameEdit.ico</disabledoff>
    <disabledon>filenameEdit.ico</disabledon>
    <activeoff>filenameEdit.ico</activeoff>
    <activeon>filenameEdit.ico</activeon>
    <selectedoff>filenameEdit.ico</selectedoff>
    <selectedon>filenameEdit.ico</selectedon>filenameEdit.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="placeholderText">
         <string>after delete the text above</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1900</width>
        <height>20</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>input the text that will be deleted </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="sizeIncrement">
         <size>
          <width>5</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="placeholderText">
         <string>originnal filename</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAdd_File"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_File">
   <property name="icon">
    <iconset>
     <normaloff>../build-FilenameEdit-Desktop_Qt_6_5_3_MSVC2019_64bit-Debug/filenameEdit.ico</normaloff>../build-FilenameEdit-Desktop_Qt_6_5_3_MSVC2019_64bit-Debug/filenameEdit.ico</iconset>
   </property>
   <property name="text">
    <string>Add File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
